webapp/sass/components/_multi-select.scss

```diff
@@ -30,11 +30,30 @@
     .Select-value-label {
         overflow: hidden;
         text-overflow: ellipsis;
-        width: calc(100% - 20px);
     }
 }
 
 .multi-select__help {
-    @include opacity(.6);
     padding: 10px 15px 0;
+
+    > div:not(.multi-select__note),
```

Comment:
What's this rule trying to accomplish? Could we give the divs and spans that are going to be affected a more descriptive name?